src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -477,7 +471,7 @@ private static void performMinimizationImpl(
         if (typesToChange.containsKey(simpleName)) {
           continue;
         }
-        if (!finder.getUsedTypeElement().contains(classFullyQualfiedName)) {
+        if (!finder.getUsedTypeElements().contains(classFullyQualfiedName)) {
```

Comment:
We should fix the spelling of `classFullyQualfiedName` --> `classFullyQualifiedName`